apps/wing/src/wingc.ts

```diff
@@ -21,9 +18,16 @@ export interface WingCompilerLoadOptions {
   imports?: Record<string, any>;
   preopens?: Record<string, string>;
   env?: Record<string, string>;
+  fs?: any;
+  wingcWASMData?: Uint8Array;
+  wingsdkManifestRoot?: string;
```

Comment:
Please add docstrings

Can `fs` be a more precise type here? What is `wingcWASMData`?